### PR TITLE
ENTER and ESC key support for inline edits / Documentation for inline edits

### PR DESCRIPTION
**Pull request by gannimet**
miscService will be renamed in the next version ;)

### CODE REVIEW

File: source/docs/app/components/inline-edits.controller.js [hunk 1]
```diff
@@ -10,19 +10,53 @@
...
-                birthDate: new Date(1989, 9, 10)
+                firstname: 'David',
+                lastname: 'Hasselhoff',
+                birthDate: new Date(1989, 9, 10),
+                deathDate: new Date(2067, 6, 14),
+                stateOfBirth: 'North Dakota',
+                stateOfDeath: 'New Hampshire',
+                alive: true,
+                dead: false,
+                favoriteArtist: {
```
Comment (bauneroni) [inline source/docs/app/components/inline-edits.controller.js:21]: Srsly?

Comment (gannimet) [inline source/docs/app/components/inline-edits.controller.js:21]: yep

File: source/kit/components/ui/miscService.js [hunk 2]
```diff
@@ -7,7 +7,10 @@
 
     function miscService() {
```
Comment (bauneroni) [inline source/kit/components/ui/miscService.js]: Make this a utilService or something, please.

File: source/kit/components/ui/miscService.js [hunk 3]
```diff
@@ -7,7 +7,10 @@
...
         return {
-            isPromise: isPromise
+            isPromise: isPromise,
+            ENTER_KEY: 13,
```
Comment (bauneroni) [inline source/kit/components/ui/miscService.js]: I'd suggest to put these in a separate constant file (keys.constants.js or whatever) using angular constants. Deal?

**Pull request merged**